lib/engine/g_1856/corporation.rb

```diff
@@ -37,7 +40,7 @@ def can_buy?
       end
 
       def can_par?
-        self == @game.national ? false : super
+        self == @game.national ? !@ipoed : super
```

Comment:
As my comment about no_buy above maybe this can be handled in general code?

Reply:
I made an Unparrable ability that prevents a corporation from being parred until the ability is removed. This doesn't help 18MEX because the NdM is parred in the private auction but it may be useful in other games